### DISCUSSION TITLE
v2/zh/guide/interface/code-theme

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#290] [Announcements]
# v2/zh/guide/interface/code-theme

一个具有强大功能的 vuepress 主题✨

https://vuepress-theme-hope.github.io/v2/zh/guide/interface/code-theme.html

**Comment by long36708** [comment]
这个代码主题功能好像不管用呀！

**Reply by Mister-Hope** [reply]
Open an issue if you think there are any isssues.

**Comment by pengdream** [comment]
shiki好像在hope里用不了，启动的时候总是报错
